templates/OME-TIFF_Conversion.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>I think this is in the wrong place - can be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>